services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -46,13 +312,71 @@ const sx = {
     marginTop: "0.5rem",
   },
   dashboardTitle: {
-    marginBottom: "1.25rem",
     fontSize: "md",
     fontWeight: "bold",
-    color: "palette.gray_medium",
+    color: "palette.gray_darkest",
   },
   // TODO: delete this
```

Comment:
🤔

Reply:
@gmrabian I tried darkest but Im not sure we have one. Dark works though.